README.md

```diff
@@ -106,6 +106,25 @@ jobs:
       uses: actions/checkout@v2
 ```
 
+#### `total length of command line and environment variables exceeds limit`
```

Comment:
I'd like to have a separate section at the bottom `## Errors` and place this under that.

Reply:
Fixed in 1b4cd2fe40d0c605e868babbff6e66f4fef5105c